pkg/models/datasource.go

```diff
@@ -207,6 +207,12 @@ type GetDataSourcesQuery struct {
 	Result []*DataSource
 }
 
+type GetDefaultDataSourceQuery struct {
+	OrgId  int64
```

Comment:
```suggestion
	OrgID  int64
```
You can rename this to conform with the Go's rule for [initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).